test/Driver/unknown-inputs.swift

```diff
@@ -11,7 +11,7 @@
 // COMPILE: 1: compile, {0}, object
 
 // RUN: %swiftc_driver -driver-print-actions %t/empty 2>&1 | %FileCheck -check-prefix=LINK-%target-object-format %s
-// RUN: not %swiftc_driver -driver-print-actions %t/empty.swiftmodule 2>&1 | %FileCheck -check-prefix=ERROR %s
+// RUN: %swiftc_driver -driver-print-actions %t/empty.swiftmodule
```

Comment:
Hm. Maybe we should warn in this case? An archive containing/referencing a single swiftmodule seems like it'd usually be a mistake.